lib/file-helper.js

```diff
@@ -24,13 +24,22 @@ const getListing = async (directoryPath) => {
 
   // Gather listing entries
   const entries = await Promise.all(basenames.map(async basename => {
-    const entryPath = join(directoryPath, basename)
+    const entryPath = normalize(join(directoryPath, basename))
 
-    return [basename, {
+    // File or directory entry
+    /** @type {DirectoryListingEntry} */
+    const entry = {
       basename,
-      path: normalize(entryPath),
+      path: entryPath,
       stats: await stat(entryPath)
-    }]
+    }
+
+    // Follow child directory listing
+    if (entry.stats.isDirectory()) {
+      entry.entries = await getListing(entryPath)
```

Comment:
This tiny change means we can remove [`recursive-readdir`](https://www.npmjs.com/package/recursive-readdir) and our `getFilesByDirectory()` helper function

Reply:
I'm very torn on that abstraction, sorry 😅 . It's technically a nice structure to represent filesystem nodes. But I'm not sure it's necessary when `glob` or `recursive-readdir` would give us a flat list of files over which to `filter`, `flatMap` in our tests:
- already tested and documented out of the box
- in a straighforward/well-known syntax, without having to `flatMap` with `listingToArray` first

The tested bit feels quite important to me there, as it's ultimately code that supports our own tests (not that a 3rd party package will be bug-proof, but they'll have tests around what that specific package does, while our helpers are more tested through our tests).

It feels to me like it makes things a bit more complex than they should be, but maybe it's just me 😊  I think I'd welcome a second opinion from someone in @alphagov/design-system-developers on that one.

Reply:
@romaricpascal You can sleep well tonight 😊 

All directory listings have now been swapped with `glob()` everywhere

Ready for review again